metadata/JapanInvoices__1__metadata.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+    "name":  "JapanInvoices",
```

Comment:
@irinel-birsan-uipath pls double check with PMs on name?
Would this be japanese invoices or JapanInvoices?

Reply:
You are right. I will double-check anyway.

Reply:
Done!